common/util.go

```diff
@@ -39,6 +39,24 @@ var (
 	ErrFormatProto = fmt.Errorf("unknown VideoProfile format for protobufs")
 	ErrFormatMime  = fmt.Errorf("unknown VideoProfile format for mime type")
 	ErrFormatExt   = fmt.Errorf("unknown VideoProfile format for extension")
+
+	mimeExts = []string{
```

Comment:
Can this be a lookup table in LPMS following the format of [1] rather than two separate lists here?

[1] https://github.com/livepeer/lpms/blob/master/ffmpeg/videoprofile.go#L76-L79